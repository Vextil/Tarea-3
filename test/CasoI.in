# ColaBinario
CrearBinario naranja 3 1 pera 8 manzana 7 anana 4 durazno 15 limon 9 FIN
CrearBinario naranja 9 2 amarillo 24 rojo 23 azul 0 violeta 8 celeste 47 rosado 15 FIN
CrearLista 1 caballo conejo gato hamster hormiga perro vaca zorro FIN
CrearBalanceado 1 3
Izquierdo 1 4
Derecho 3 5
Linealizar 1 2
AlturaCantidadBinario 2
Encolar 2
Encolar 1
Encolar 4
Encolar 3
Desencolar
Encolar 5
Desencolar
Desencolar
# Solicitudes
IngresarSolicitudes mesa 6 silla 44 puerta 7 ventana 83 pared 5 sillon 6 banco 44 banco 7 FIN
CancelarSolicitud
EnAlgunoSolicitudes 6 44 1
EnAmbosSolicitudes 7 44 2
SoloEnUnoSolicitudes 7 44 3
ListarSolicitudes
# Set
ConstruirSet 1 9
CopiarSet 2 4
UnionInterDif 1 2 5 6 7 8
InsertarEnSet 5 2
RemoverDeSet 4 1
PerteneceASet 4 2
PerteneceASet 5 1
PerteneceASet 5 2
PerteneceASet 6 1
PerteneceASet 6 2
PerteneceASet 7 1
PerteneceASet 7 2
Fin
