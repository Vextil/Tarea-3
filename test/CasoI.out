1># ColaBinario
2>OK
3>OK
4>OK
5>OK
6>OK
7>OK
8>OK
9>Altura: 4; Cantidad: 7
10>OK
11>OK
12>OK
13>OK
14>
   (8,violeta)
    (15,rosado)
  (23,rojo)
 (9,naranja)
    (47,celeste)
   (0,azul)
  (24,amarillo)

15>OK
16>
  (8,pera)
 (3,naranja)
  (7,manzana)
     (9,limon)
    (15,durazno)
   (4,anana)

17>
 (7,manzana)
    (9,limon)
   (15,durazno)
  (4,anana)

18># Solicitudes
19>OK
20> 5
21>OK
22>OK
23>OK
24>(6,mesa) (6,sillon) (7,puerta) (7,banco) (44,silla) (44,banco) (83,ventana) 
25># Set
26>OK
27>OK
28>OK
29>OK
30>OK
31>NO NO NO NO NO NO 
32>NO NO NO NO NO NO NO NO 
33>SI SI SI SI SI SI 
34>NO NO NO NO NO NO NO NO 
35>NO NO NO NO NO NO 
36>NO NO NO NO NO NO NO NO 
37>NO NO NO NO NO NO 
38>Fin
